docs/sources/next/javascript-api/k6-experimental/browser/element-handle/_index.md

```diff
@@ -19,7 +19,7 @@ weight: 04
 | <a href="https://playwright.dev/docs/api/class-elementhandle#element-handle-check" target="_blank" >elementHandle.check([options])</a>                                       | -                                |
 | <a href="https://playwright.dev/docs/api/class-elementhandle#element-handle-click" target="_blank" >elementHandle.click([options])</a>                                       | -                                |
 | <a href="https://playwright.dev/docs/api/class-elementhandle#element-handle-content-frame" target="_blank" >elementHandle.contentFrame()</a>                                 | -                                |
-| <a href="https://playwright.dev/docs/api/class-elementhandle#element-handle-dblclick" target="_blank" >elementHandle.dblclick([options])</a>                                 | -                                |
+| [elementHandle.dblclick([options])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/element-handle/dblclick)                                 | Double click on the element.     |
```

Comment:
The dash.

```suggestion
| [elementHandle.dblclick([options])](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/elementhandle/dblclick)                                 | Double click on the element.     |
```

Reply:
Resolved in 38f49cf5289dc1563eadf6c76079c26e00254772